app/installer/views/installer.php

```diff
@@ -123,6 +124,45 @@
                                     </div>
                                 </div>
                             </div>
+                            <div class="uk-form-row" v-if="config.database.default == 'pgsql'">
+                                <div class="uk-form-row">
+                                    <label for="form-mysql-dbhost" class="uk-form-label">{{ 'Hostname' | trans }}</label>
+                                    <div class="uk-form-controls">
+                                        <input id="form-mysql-dbhost" class="uk-width-1-1" type="text" name="host" value="localhost" v-model="config.database.connections.mysql.host" v-validate:required>
```

Comment:
It feels off that the `.mysql` configuration variables are used. We should either rename that to something more general or use separate configuration variables?